.github/workflows/pr.yaml

```
@@ -25,7 +25,8 @@ jobs:
       - docs-build
       - wheel-build-cudf
       - wheel-tests-cudf
-      - test-cudf-polars
+      - wheel-build-cudf-polars
+      - test-wheel-cudf-polars
```

Match the existing convention.
```suggestion
      - wheel-tests-cudf-polars
```